Random `testFileTimes` test failure in `test_win32file.py`

This isn't the first time it happens to me, but is very rare. I haven't looked into it yet, but probably related to time precision, or obtaining "current time" twice leading to a few ticks of difference.

https://github.com/mhammond/pywin32/actions/runs/8283474275/job/22666727095?pr=2145#step:6:292
```
FAIL: testFileTimes (test_win32file.TestSimpleOps)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\pywin32\pywin32\win32\test\test_win32file.py", line 234, in testFileTimes
    self.assertTrue(now <= ct <= nowish, (now, ct))
AssertionError: False is not true : (datetime.datetime(2024, 3, 14, 15, 43, 53, tzinfo=TimeZoneInfo('GMT Standard Time', True)), pywintypes.datetime(2024, 3, 14, 15, 43, 54, 17000, tzinfo=TimeZoneInfo('GMT Standard Time', True)))
```

This is a different issue than https://github.com/mhammond/pywin32/issues/2080

--- a/win32/test/test_win32file.py
+++ b/win32/test/test_win32file.py
@@ -227,16 +227,25 @@ class TestSimpleOps(unittest.TestCase):
         )
         try:
             ct, at, wt = win32file.GetFileTime(f)
-            self.assertTrue(
-                ct >= now,
+            # NOTE (Avasam): I've seen the time be off from -0.003 to +1.11 seconds,
+            # so the above comment about microseconds might be wrong.
+            # Let's standardize ms and avoid random CI failures
+            # https://github.com/mhammond/pywin32/issues/2203
+            ct = ct.replace(microsecond=0)
+            at = at.replace(microsecond=0)
+            wt = wt.replace(microsecond=0)
+            self.assertGreaterEqual(
+                ct,
+                now,
                 f"File was created in the past - now={now}, created={ct}",
             )
-            self.assertTrue(now <= ct <= nowish, (now, ct))
-            self.assertTrue(
-                wt >= now,
+            self.assertTrue(now <= ct <= nowish, (now, ct, nowish))
+            self.assertGreaterEqual(
+                wt,
+                now,
                 f"File was written-to in the past now={now}, written={wt}",
             )
-            self.assertTrue(now <= wt <= nowish, (now, wt))
+            self.assertTrue(now <= wt <= nowish, (now, wt, nowish))
 
             # Now set the times.
             win32file.SetFileTime(f, later, later, later, UTCTimes=True)

